### ISSUE TITLE
Improve defense against mobile editor accessibility regressions

### ISSUE THREAD

**Issue by dcalhoun** [WordPress/gutenberg#54361]
## What problem does this address?

The mobile Gutenberg project is vulnerable to accessibility (a11y) regressions as we lack a well defined testing strategy — manual or automated. 

## What is your proposed solution?

Document testing approaches and set up tooling to help us better protect the project from a11y regressions. 

- [ ] Integrate [`eslint-plugin-react-native-a11y`](https://github.com/FormidableLabs/eslint-plugin-react-native-a11y). 
- [ ] Document high-level information/resources for testing iOS and Android a11y features and text entry. E.g. common approaches for text entry/navigation while using assistive technologies. 
- [ ] Capture a set of foundational a11y tests to perform during releases and major dependency upgrades. 
- [ ] Research opportunities to automate the aforementioned foundational a11y tests. 
- [ ] Explore the opportunity to automate a11y assertions with [`react-native-accessibility-engine`](https://github.com/aryella-lacerda/react-native-accessibility-engine). 
- [ ] Research leveraging iOS' [`performAccessibilityAudit`](https://developer.apple.com/documentation/xctest/xcuiapplication/4191487-performaccessibilityaudit) to automate a11y tests.
- [ ] Research leveraging Android's [`AccessibilityChecks`](https://developer.android.com/guide/topics/ui/accessibility/testing#automated) to automate a11y tests.

**Comment by tyxla** [comment]
This is no longer relevant after the React Native implementation was removed in #78747. Closing.
